Sources/HTTP/Responder/HTTPScheme.swift

```diff
@@ -2,16 +2,16 @@
 public struct HTTPScheme {
     /// Plaintext data over TCP. Uses port `80` by default.
     public static var http: HTTPScheme {
-        return .init(80) { .done(on: $0.eventLoop) }
+        return .init(80) { channel, _ in .done(on: channel.eventLoop) }
     }
 
     /// Enables TLS (SSL). Uses port `443` by default.
     public static var https: HTTPScheme {
-        return .init(443) { channel in
+        return .init(443) { channel, serverHostname in
```

Comment:
I think it could cut down on code dupe to do the `if scheme.defaultPort == 443 && !hostname.isIPAddress()` logic here and just always pass in the hostname. (`String` not `String?`)